tools/cockpit.spec

```diff
@@ -324,8 +324,10 @@ Summary: Cockpit Web Service
 Requires: glib-networking
 Requires: openssl
 Requires: glib2 >= 2.50.0
-Requires: (selinux-policy >= %{_selinux_policy_version} if selinux-policy-%{selinuxtype})
-Requires(post): (policycoreutils if selinux-policy-%{selinuxtype})
+Requires: (%{name}-selinux-policies if selinux-policy-base)
+# Split SELinux policies to subpackage
+Obsoletes: %{name}-ws < 337-1
+Conflicts: %{name}-ws < 337-1
```

Comment:
TF tests fail with
```
Error: 
 Problem: package cockpit-selinux-policies-336-1.20250422125529379097.pr21902.115.gfe057bd66.el9.x86_64 from @commandline conflicts with cockpit-ws < 337-1 provided by cockpit-ws-336-1.20250422125529379097.pr21902.115.gfe057bd66.el9.x86_64 from @commandline
  - conflicting requests
```

We released 337 in the meantime, but now of course this conceptually would have to be < 338. This is a bit tricky for a future release number. To cover git builds we could at least temporarily use `< 337-1.2025` ?

Reply:
Also, these conflicts shouldn't be *here* but against c-selinux-policies. cockpit-ws shoudl never obsoletes/conflicts itself.

Reply:
Fixed both. Let's give them a try on CI.